frontend/lib/ui/artefact_page/test_execution_expandable.dart

```diff
@@ -13,43 +13,56 @@ class TestExecutionExpandable extends ConsumerWidget {
 
   final TestExecution testExecution;
 
-  @override
-  Widget build(BuildContext context, WidgetRef ref) {
+  Row getTestExecutionTileTitle(BuildContext context) {
```

Comment:
small thing, but perhaps this could be a widget on it's own

Reply:
Good idea, I converted into a small private widget.